app/views/activities/index.html.erb

```diff
@@ -106,16 +106,27 @@ See COPYRIGHT and LICENSE files for more details.
       <legend class="form--fieldset-legend"><%= t(:description_filter) %></legend>
     <p>
       <%= hidden_field_tag "event_types[]" %>
-      <% @activity.event_types.sort_by { |type| t("label_#{type.singularize}_plural") }.each do |t| %>
-        <%= check_box_tag "event_types[]", t, @activity.scope.include?(t), id: "event_types_#{t}" %>
-        <label for="event_types_<%=t%>"><%=t("label_#{t.singularize}_plural")%></label>
+      <% @activity.event_types.sort_by { |type| t("label_#{type.singularize}_activity_filter") }.each do |t| %>
+        <% unless t == "project_attributes" %>
```

Comment:
I wonder if the `@activity.event_types.sort_by { |type| t("label_#{type.singularize}_activity_filter") }` on the line above could not be replaced by something else like `@activity.module_filters`. It would return the list of types excluding the `"project_attributes"` one, making the check useless. It could also handle the alphabetical sort to remove some code from the view.

Reply:
I'll leave that for later.